### ISSUE TITLE
The problem of Loss equal to nan

### ISSUE THREAD

**Issue by lincgcg** [shashadehuajiang/trace_classifier#7]
We can obtain normal results when reproducing the "LSTM+ATT" setting, but when setting PACKET2FLOW in config.py to other values, nan occurs.

We guess it is a learning rate problem, but the paper does not give the learning rate for each PACKET2FLOW setting.

Can you provide specific hyperparameter settings when PACKET2FLOW and FLOW2TRACE take different values?

**Comment by shashadehuajiang** [comment]
Well, In our experiments, the LEARNING_RATE ranged approximately from 2e-4 to 1e-3. However, as far as I remember, it seems that the loss never encountered a situation where it became 'nan', so I'm not sure what the specific reason might be.
